src/cjs.js

```diff
@@ -1 +1,4 @@
-module.exports = require('./index').default;
+const fileLoader = require('./index');
+
+module.exports = fileLoader.default;
+module.exports.raw = fileLoader.raw;
```

Comment:
@Kovensky Perhaps, but on the other hand, I would not want to add workarounds to the loader code, because it can be misleading. In any case, I'm open to ideas, I'll be glad to see another solution